src/DeveloperModeGPSCheck.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+
+import { checkStations } from 'stations';
+
+export const noLocation = 'noLocation';
+export const invalidLocation = 'invalidLocation';
+export const validLocation = 'validLocation';
+export const locationNotAvailable = 'locationNotAvailable';
+export const locationTimerOut = 'locationTimerOut';
+
+const styles = () => ({
+  card: { minWidth: 275 },
+  lowered: { marginTop: 12 },
+});
+
+function getCorrectLocationQuestion(whichQuestion) {
+  if (whichQuestion === noLocation) {
+    return 'Välkommen till världens bästa spel!';
+  }
+  if (whichQuestion === locationNotAvailable) {
+    return 'Geolocation is not supported by this browser. För att kunna spela måste du ha checkat in i spelet vid en hållplats under de senaste 15 minutrarna.';
+  }
+  if (whichQuestion === 'locationTimerOut') {
+    return 'Du har varit bort från en hållplats förlänge! Gå till en för att forstätta spela!';
+  }
+  return 'Du kan tyvärr inte spela världens bästa spel. För att kunna spela måste du ha checkat in i spelet vid en hållplats under de senaste 15 minutrarna. Gå till en hållplats och försök igen!';
+}
+
+function DeveloperModeGPSCheck({
+  classes: { card, lowered },
+  locationCheck,
+  locationOk,
+}) {
+  return (
+    <Card className={card}>
+      <CardContent>
+        <Typography variant="h6" className={lowered}>
+          {getCorrectLocationQuestion(locationOk)}
+        </Typography>
+      </CardContent>
+      <Divider />
+      <CardContent>
+        <Typography variant="h9" className={lowered}>
```

Comment:
h9 is not a variant, this might cause undefined bahaviour. There's a list of available variants in the [Material UI docs](https://material-ui.com/style/typography/#component).